lzero/model/unizero_world_models/world_model.py

```diff
@@ -546,7 +665,10 @@ def _process_obs_act_combined_cont(self, obs_embeddings_or_act_tokens, prev_step
             obs_act = torch.cat([obs, act], dim=1)
             obs_act_embeddings[:, i * (K + 1):(i + 1) * (K + 1), :] = obs_act
 
-        return obs_act_embeddings + self.pos_emb(prev_steps + torch.arange(num_steps, device=self.device)), num_steps
+        return_result = obs_act_embeddings
+        if not self.config.rotary_emb:
+            return_result += self.pos_emb(prev_steps + torch.arange(num_steps, device=self.device))
```

Comment:
you don't need to add this if branch here

Reply:
The polishing of pos_emb will be addressed in a future PR.